lms/static/styles/components/_Dialog.scss

```diff
@@ -26,10 +20,13 @@
 }
 
 .Dialog__content {
+  display: flex;
```

Comment:
moving the display flex here and setting the max height to 90vh allows for the dialogs to fill up the screen if they need to, but not beyond.